gp-google-sheets/gpgs-process-feed-on-partial-entry-save.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Gravity Perks // GP Google Sheets // Process Feed when Partial Entry is Saved.
+ *
+ * This snippet processes google sheet feed for partial entries as well.
+ *
+ */
+add_action( 'gform_partialentries_post_entry_saved', 'send_to_google_sheet_on_partial_entry_saved', 10, 2 );
+function send_to_google_sheet_on_partial_entry_saved( $partial_entry, $form ) {
+	if ( function_exists( 'gp_google_sheets' ) ) {
+		$partial_entry['status'] = 'partial';
```

Comment:
I see this is in the sample snippet as well. Do you understand why it's needed?

Reply:
@spivurno The only thing I noticed was if I did not set the 'status' for the partial entry, there would be a warning popping up:

`PHP Warning:  Undefined array key "status" in /.../class-gf-feed-addon.php` at [this code block ](https://github.com/gravityforms/gravityforms/blob/master/includes/addon/class-gf-feed-addon.php#L296-L302).

The created partial entry via GF Partial Entries has it's 'status' value not set, so it just made sense to let it set to 'partial' (following Gravity Forms examples), instead of leaving it unset (PHP warning), or setting it to other values like 'active'.

And yes, even with the warning the Google Sheets feed processed just fine. 

Do you have any alternate suggestions for this?

Reply:
Ah, ok. Nope! I was just curious. 😄